dts/bindings/sensor/memsic,mc3419.yaml

```diff
@@ -21,3 +21,30 @@ properties:
       has two interrupt pins.By default the interrupt are routed
       to interrupt pin 1, by enabled this property interrupt are
       routed to interrupt pin 2.
+
+  lpf-fc-sel:
+    type: int
+    default: 0
+    enum:
+      - 0
+      - 9
+      - 10
+      - 11
+      - 13
+    description: |
+      Select lpf cut-off frequency if lpf-en is true. otherwise this has no effect
+      Possible values and corresponding values are listed bellow
```

Comment:
```suggestion
      Select lpf cut-off frequency if lpf-en is true. otherwise this has no effect
      Possible values and corresponding values are listed bellow.
      Default is disabling LPF.
```